ALERTS.md

```diff
@@ -0,0 +1,35 @@
+# Contributing Guide
+
+- [Introduction](#introduction)
+- [Configuration](#configuration)
+- [Slack Web API](#slack-web-api)
+- [Error Message](#error-message)
+
+## Introduction
+
+The reason monitoring tests are separated by goalie is so that, in case of an incident, the person on-call can find the failing test easily. But how can they know when a test is failing unless they wade into the GitHub actions page and manually search for a failing test? That's where alerts come in.
```

Comment:
This paragraph just doesn't work for me. Sorry. In keeping with your tone, how about

"If a contract test fails during a monitoring run, the monitor sends an alert to the Slack channel for the relevant on-call goalie. The rest of this guide explains the previous sentence."

Reply:
That's much more concise, thank you! I wasn't really sure how to phrase the intro, so this helped.